python/src/scipp/plot/__init__.py

```diff
@@ -3,62 +3,57 @@
 # @file
 # @author Neil Vaytet
 
-import importlib
+from .plot import plot as _plot
 
 # If we are running inside a notebook, then make plot interactive by default.
 # From: https://stackoverflow.com/a/22424821
+# try:
+is_doc_build = False
 try:
-    import matplotlib
-    is_doc_build = False
-    try:
-        from IPython import get_ipython
-        ipy = get_ipython()
-        if ipy is not None:
+    from IPython import get_ipython
+    import matplotlib as mpl
+    ipy = get_ipython()
+    if ipy is not None:
+
+        # Check if a docs build is requested in the metadata. If so,
+        # use the default Qt/inline backend.
```

Comment:
Not for now, but would it make sense to use some sort of mock view for testing?

Reply:
Yes, still completely unclear how we should set up testing for all this